nexus/reconfigurator/planning/src/blueprint_builder.rs

```diff
@@ -868,14 +874,12 @@ impl<'a> BlueprintZones<'a> {
                     // If it's not there either, then this must be a new sled
                     // and we haven't added any zones to it yet.  Use the
                     // standard initial config.
-                    .unwrap_or_else(|| OmicronZonesConfig {
+                    .unwrap_or_else(|| BlueprintZonesConfig {
                         generation: Generation::new(),
                         zones: vec![],
                     });
 
-                // This is not strictly necessary.  But for testing, it's
-                // helpful for things to be in sorted order.
```

Comment:
Can we keep this comment?

Reply:
Put back the part of the comment starting with "For testing..." -- left the full comment on the doc comment.